build.py

```diff
@@ -120,22 +134,24 @@
 
 # Check that our environment contains everything we need to do a build.
 
-for envVar in ( "GITHUB_RELEASE_TOKEN", ) :
-	if envVar not in os.environ	:
-		parser.exit( 1,  "{0} environment variable not set".format( envVar ) )
+if args.upload :
+	if "GITHUB_RELEASE_TOKEN" not in os.environ	:
+		parser.exit( 1,  "GITUHB_RELEASE_TOKEN environment variable not set" )
```

Comment:
Add a check that Arnold and 3Delight root are set, and exit if not. That way we can't accidentally make a release build without full renderer support.

Reply:
Added in https://github.com/GafferHQ/build/pull/10/commits/683e03b84fcbb9f69ba56155d09f42aaa0903962